src/main/java/it/eng/idsa/businesslogic/service/impl/BrokerServiceImpl.java

```diff
@@ -64,7 +64,7 @@ public void sendBrokerRequest(Message message, String payload) {
 			
 			response = sendDataToBusinessLogicService.sendMessageBinary(brokerURL, multipartMessage, headers, true);
```

Comment:
We should consider to close this response (according to documentation
Call call = client.newCall(request);
   try (Response response = call.execute()) {
     ... // Use the response.
   }